### PR TITLE
Questions for Shared Preferences, Sqlite and Push Notifications

### PR DESCRIPTION
**Pull request by FaisalAbid**
it's important to have these links in this documentation as these are the most common questions people will ask and not necessarily go looking on pub.dartlang

I thought about writing more on these, but the ReadMe's of the plugins do a good job and the writeup would end up being redundant.

### CODE REVIEW

File: flutter-for-android.md [hunk 1]
```diff
@@ -2019,3 +2019,59 @@ class SampleApp extends StatelessWidget {
   }
 }
 {% endprettify %}
+
+
+# Databases
```
Comment (sethladd) [inline flutter-for-android.md]: nit: newlines after headers

Comment (sethladd) [inline flutter-for-android.md]: "Databases and local storage" ?

File: flutter-for-android.md [hunk 2]
```diff
@@ -2019,3 +2019,59 @@ class SampleApp extends StatelessWidget {
...
+  runApp(new MaterialApp(
+    home: new Scaffold(
+      body: new Center(
+      child: new RaisedButton(
+        onPressed: _incrementCounter,
+        child: new Text('Increment Counter'),
+        ),
+      ),
+    ),
+  ));
```
Comment (sethladd) [inline flutter-for-android.md]: missing trailing comma here

File: flutter-for-android.md [hunk 3]
```diff
@@ -2019,3 +2019,59 @@ class SampleApp extends StatelessWidget {
...
+
+{% endprettify %}
+
+## How do I access SQLite in Flutter?
+In Android, you would use SQLite to store structured data that you can query 
+via SQL.
+
+In Flutter, you can access this functionality by using the SQFlite plugin 
+[SQFlite](https://pub.dartlang.org/packages/sqflite)
+
```
Comment (sethladd) [inline flutter-for-android.md]: nite: remove extra blank line

File: flutter-for-android.md (hunk 1 above)
Comment (sethladd) [reply flutter-for-android.md]: global comment

**Comment by sethladd** [comment]
Great! Minor copy comments

File: flutter-for-android.md (hunk 2 above)
Comment (FaisalAbid) [reply flutter-for-android.md]: Let me know if this is good, I ran the formatter on it in intellij!

File: flutter-for-android.md (hunk 3 above)
Comment (FaisalAbid) [reply flutter-for-android.md]: screen shot 2017-11-01 at 9 58 59 am

I've removed this line but Git is not reflecting it, I think the editor might be adding an invisible line

**Comment by FaisalAbid** [comment]
lmk if this is good to merge! then we should be good to promote this guy!

**Comment by sethladd** [comment]
lgtm! sorry for the delay

**Pull request merged**